MyTasks/Task2_1/UserAndBotDialogue.cs

```diff
@@ -0,0 +1,60 @@
+﻿
+Console.WriteLine("Здравствуйте! Введите Ваше имя:");
+var userName = Console.ReadLine();
+
+while (char.IsLower(userName, 0))
+{
+    Console.WriteLine("Пожалуйста, введите имя с заглавной буквы:");    // "обработка" неправильного пользовательского ввода
+    userName = Console.ReadLine();
+}
```

Comment:
It does not fit the task. Need a transformation, not a validation. Read the task
Don't force the user to type a million times his name

Reply:
Done.